x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -190,34 +156,20 @@ export const uiSettings: Record<string, UiSettingsParams<boolean | number | stri
     }),
     schema: schema.number({ min: 1 }),
   },
-  [apmTraceExplorerTab]: {
-    category: [observabilityFeatureId],
-    name: i18n.translate('xpack.observability.apmTraceExplorerTab', {
-      defaultMessage: 'APM Trace Explorer',
-    }),
-    description: i18n.translate('xpack.observability.apmTraceExplorerTabDescription', {
-      defaultMessage:
-        '{technicalPreviewLabel} Enable the APM Trace Explorer feature, that allows you to search and inspect traces with KQL or EQL',
-      values: { technicalPreviewLabel: `<em>[${technicalPreviewLabel}]</em>` },
-    }),
-    schema: schema.boolean(),
-    value: false,
-    requiresPageReload: true,
-    type: 'boolean',
-  },
-  [apmOperationsTab]: {
+
+  [apmLabsButton]: {
     category: [observabilityFeatureId],
-    name: i18n.translate('xpack.observability.apmOperationsBreakdown', {
-      defaultMessage: 'APM Operations Breakdown',
+    name: i18n.translate('xpack.observability.apmLabsButton', {
+      defaultMessage: 'Labs mode',
     }),
-    description: i18n.translate('xpack.observability.apmOperationsBreakdownDescription', {
+    description: i18n.translate('xpack.observability.apmLabsButtonDescription', {
       defaultMessage:
-        '{technicalPreviewLabel} Enable the APM Operations Breakdown feature, that displays aggregates for backend operations',
-      values: { technicalPreviewLabel: `<em>[${technicalPreviewLabel}]</em>` },
+        'Turn on for automatically opt-in to future technical preview features released.',
```

Comment:
What about this?

> When enabled, provides access to the technical preview Labs features for APM.

Reply:
@bmorelli25 I change the title and description of this option. For now it's not possible to automatically opt-in for future features. This is the new value: https://github.com/elastic/kibana/pull/139553/commits/08e37089297b1f18f083faa73bac991b2caefef8